internal/ingress/controller/store/store.go

```diff
@@ -173,7 +173,7 @@ func (i *Informer) Run(stopCh chan struct{}) {
 	if !cache.WaitForCacheSync(stopCh,
 		i.Ingress.HasSynced,
 	) {
-		runtime.HandleError(fmt.Errorf("Timed out waiting for caches to sync"))
+		runtime.HandleError(fmt.Errorf("timed out waiting for caches to sync"))
```

Comment:
The capital letter is by design here. See [logError](https://github.com/kubernetes/apimachinery/blob/master/pkg/util/runtime/runtime.go#L112-L115)